RELEASE_NOTES.md

```diff
@@ -1,120 +1,136 @@
 ### OVERVIEW
 
-The Narrative Interface allows users to craft KBase Narratives using a combination of GUI-based commands, Python and R scripts, and graphical output elements.
+The Narrative Interface allows users to craft KBase Narratives using a combination of GUI-based commands, Python and R
```

Comment:
What happened here? Is there a markdown linter we should be using / enforcing now?